aiida_mlip/calculations/train.py

```diff
@@ -148,6 +169,13 @@ def prepare_for_submission(
 
             # Update the config file with absolute paths
             config_parse = config_parse.replace(mlip_dict[file], str(abs_path))
+
+        # Add foundation_model to the config file if fine-tuning is enabled
+        if self.inputs.fine_tune and "model" in self.inputs:
```

Comment:
Just to check, is `self.inputs.fine_tune` always defined here (I'm guessing yes if it's defined via the default `spec.input`)?